llama-index-core/llama_index/core/base/llms/base.py

```diff
@@ -253,3 +254,19 @@ async def astream_complete(
                 print(response.text, end="", flush=True)
             ```
         """
+
+    def __init_subclass__(cls, **kwargs) -> None:
```

Comment:
Lets add a docstring to this method to explain what's happening in here for future devs